content/home/hero.md

```diff
@@ -36,13 +36,14 @@ hero_media = "examtools_logo_ss.svg"
 # Call to action links (optional).
 #   Display link(s) by specifying a URL and label below. Icon is optional for `[cta]`.
 #   Remove a link/note by deleting a cta/note block.
-[cta]
-  url = "start/"
-  label = "Get Started"
-  icon_pack = "fas"
-  icon = "download"
+
```

Comment:
My only thought here was that it might be nice to have a single "getting started" page which walks them through getting set up in addition to the documentation; thoughts?

Reply:
My thinking is to get the documentation done, then we can go back adding pages like "Getting Started".